converter/convert-to-ttl.rb

```diff
@@ -49,7 +49,7 @@ def bestuursorgaan_voor_gemeentenaam(naam, type, date)
           PREFIX org: <http://www.w3.org/ns/org#>
           PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
           PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
-          SELECT ?iri
+          SELECT DISTINCT ?iri
```

Comment:
i added this distinct everywhere in the select queries. This is because i wanted to start from an existing database of a semantic.works application, that sometimes has the same triple in different graphs